### PR TITLE
SDK work to support featured drops

### PR DESCRIPTION
**Pull request by ewno**

### CODE REVIEW

File: Sources/vault-swift/vault_swift.swift [hunk 1]
```diff
@@ -51,10 +51,53 @@ public class ProteusWallet {
...
+            }
+        }
+        
+        return featuredDrops
+    }
     /**
      Returns the ID of the claim transaction
      */
-    public func claim(_ nft: DecryptableNFT, with verificationCode: String? = nil) async throws -> String {
+    public func claim(_ mintAddress: String, with verificationCode: String? = nil) async throws -> String {
```
Comment (andyboyd) [inline Sources/vault-swift/vault_swift.swift:100]: What was the reason for this change? `listClaimableNFTs` gives back an array of `DecryptableNFT` objects, and those should be the only things able to be claimed.

File: Sources/vault-swift/Models/DTOs/JsonCreatorProfileExt.swift [hunk 2]
```diff
@@ -30,5 +30,5 @@ struct JsonCreatorProfileExt: Decodable {
     let verified: Bool
     let image: String
     let description: String
-    let properties: Properties
+    let properties: Properties?
```
[the remark this answers is not in the document]
Comment (ewno) [reply Sources/vault-swift/Models/DTOs/JsonCreatorProfileExt.swift:33]: Back end (Ant) has confirmed this should be optional.

File: Sources/vault-swift/Models/DTOs/JsonFeaturedDropExt.swift [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+//  Created by Andrew Noble on 02/12/2022.
+//
+
+import Foundation
+
+struct JsonFeaturedDropExt: Decodable {
+    
+    struct Variant: Decodable {
+        let candyMachine: String
+        let priceUsdCents: String
```
[the remark this answers is not in the document]
Comment (ewno) [reply Sources/vault-swift/Models/DTOs/JsonFeaturedDropExt.swift:14]: I'll raise a ticket for back end to look at this.

File: Sources/vault-swift/Models/DTOs/JsonFeaturedDropExt.swift [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    }
+    
+    struct Creator: Decodable {
+        let profile: String?
+        let percentageShare: Int
+    }
+    
+    struct FeaturedDropData: Decodable {
+        let creators: [Creator]
+        let startDate: String
```
[the remark this answers is not in the document]
Comment (ewno) [reply Sources/vault-swift/Models/DTOs/JsonFeaturedDropExt.swift:26]: And I'll raise a ticket for back end to look at this.

File: Sources/vault-swift/vault_swift.swift (hunk 1 above)
Comment (andyboyd) [reply Sources/vault-swift/vault_swift.swift:100]: OK let's stick with this for now, with an eye on improving things later

**Pull request merged**